src/adv.py

```diff
@@ -36,9 +35,9 @@
 #
 # Main
 #
-
+print("Welcome to adventure time!")
 # Make a new player object that is currently in the 'outside' room.
-
+Player = Player(input("Name: "), room['outside'])
```

Comment:
Great! Please note that this instance may be better lower-cased. It can become hard to see what is an instance of the class and what is the class itself if the instance is capitalized like the class. `player = Player...` may be better.